pep-0648.rst

```diff
@@ -0,0 +1,300 @@
+PEP: 0648
+Title: Extensible customizations of the interpreter at startup
+Author: Mario Corchero <mariocj89@gmail.com>
+Sponsor: Pablo Galindo
+BDFL-Delegate: XXXX
+Discussions-To: https://discuss.python.org/t/pep-648-extensible-customizations-of-the-interpreter-at-startup/6403
+Status: Draft
+Type: Standards Track
+Content-Type: text/x-rst
+Created: 30-12-2020
+Python-Version: 3.11
```

Comment:
You want to target 3.11? You are still on time for 3.10 if you want to change this